index.html

```diff
@@ -135,5 +135,24 @@ <h2>About <br>Myself</h2>
             <li class="about-skill-list border">Skills <img src="./img/chevron-right.png" alt="chevron" class="chevron"></li>
         </ul>
     </section>
+    <!-- Contact me section-->
+    <section class="contact-form">
+        <div class="background-img">
+            <!-- Adding background image through css -->
+        </div>
+        <h2 class="contact-me">Contact me</h2>
+        <!--support message-->
+        <p class="support-msg">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+        </p>
+        <form action="https://formspree.io/f/mjvzkgvn" method="POST" class="form">
+            <input type="text" name="user_name" placeholder="Florin" maxlength="30" required>
+            <input type="email" name="user_email" placeholder="vflorin1995@gmail.com" required>
```

Comment:
- [ ] It will be better if you add a good placeholder instead of your name and email to not confuse the recruiter when trying to connect with you.
```
Good example:
name: your name
email: your email
```